gravitee-apim-console-webui/src/management/api/history-v4/api-history-v4.component.html

```diff
@@ -0,0 +1,35 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<mat-card>
+  <mat-card-content>
+    <div class="history">
+      <div class="history__header">
+        <h3 class="history__header__title">API version history</h3>
+        <p>Keep track of every API's changes per deployment</p>
+      </div>
+    </div>
+  </mat-card-content>
+  <ng-container *ngIf="apiEvents$ | async as apiDeployments">
```

Comment:
I've noticed that you've used as many async pipes as possible, which is really good practice 💪